### DISCUSSION TITLE
Code Server on tiny core

### DISCUSSION THREAD

**Discussion by blazstefin** [coder/code-server#4989] [Q&A] [answered]
Hi
Will code server work on Tiny Core OS?
Thank you

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
I guess so, if Tiny Core is based on Debian or RHEL. Then the package manager can install the binary onto your system.

**Comment by lg-nineoneninethreenine** [comment]
Yeah, just run this on Terminal:
```
curl -fsSL https://code-server.dev/install.sh | sh
```
Then restart your shell for a small refresh; best to do that to get updated binaries and then just start code-server in port 8080.
